src/translations/TranslationListItem.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import TextField from '@material-ui/core/TextField';
+import Fab from '@material-ui/core/Fab';
+import PlayIcon from '@material-ui/icons/PlayArrowOutlined';
+import ApiUtils from '../utils/ApiUtils';
+import './TranslationListItem.css';
+
+class TranslationListItem extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.handleTranslationChange_ = this.handleTranslationChange_.bind(this);
```

Comment:
Internal TS style prefers `() => {this.foo()}` to `foo.bind(this)`, but either is fine.

Reply:
Updated.